outrank/algorithms/importance_estimator.py

```diff
@@ -57,17 +59,22 @@ def sklearn_surrogate(
     unique_values, counts = np.unique(vector_second, return_counts=True)
 
     # Establish min support for this type of ranking.
-    if counts[0] < len(unique_values) * (2**5):
-        estimate_feature_importance = 0
+    # if counts[0] < len(unique_values) * (2**5):
+    #     estimate_feature_importance = 0
 
-    else:
+    if X.shape[0] == 0 and X.shape[1] == 0:
         vector_first = transf.fit_transform(vector_first.reshape(-1, 1))
         estimate_feature_importance_list = cross_val_score(
             clf, vector_first, vector_second, scoring='neg_log_loss', cv=4,
         )
-
-        estimate_feature_importance = 1 + \
-            np.median(estimate_feature_importance_list)
+    else:
+        X = np.concatenate((X,vector_first.reshape(-1, 1)), axis=1)
```

Comment:
There is a space missing after `X`, wondering why lint didn't catch that. @miha-jenko maybe some idea?